.github/workflows/python-sdk-publish.yaml

```diff
@@ -0,0 +1,39 @@
+name: "Python: Publish python/serverless-sdk"
+
+on:
+  push:
+    tags:
+      - "python/serverless-sdk@[0-9]+.[0-9]+.[0-9]+"
+
+jobs:
+  publishNewSdkVersion:
+    name: Publish new version
+    runs-on: ubuntu-latest
+    steps:
+      - name: Install Python and Pip
+        uses: actions/setup-python@v4
+        with:
+          python-version: '3.7'
+
+          # ensure project dependencies are cached
+          # When using only `pyproject.toml` for dependencies, see:
+          #  https://github.com/actions/setup-python/issues/529#issuecomment-1367029699
+          cache: 'pip'
+          cache-dependency-path: |
+            **/pyproject.toml
+
+      - name: Install main project dependencies
+        run: |
+          cd python/packages/sdk
+          python3 -m pip install .
```

Comment:
This will ensure that build dependencies are installed before the package is built and published. Otherwise the build dependencies will need to be explicitly installed in the publication step, and the two can go out of sync if new build deps are added to the package.